readme.md

```diff
@@ -187,6 +187,7 @@ See [FAQ Archive](https://github.com/sindresorhus/pure/wiki/FAQ-Archive) for pre
 	- [therealklanni/purity](https://github.com/therealklanni/purity) - More compact current working directory, important details on the main prompt line, and extra Git indicators.
  	- [intelfx/pure](https://github.com/intelfx/pure) - Solarized-friendly colors, highly verbose, and fully async Git integration.
 	- [dfurnes/purer](https://github.com/dfurnes/purer) - Compact single-line prompt with built-in Vim-mode indicator.
+	- [chabou/pure-now](https://github.com/chabou/pure-now) - Maintained fork with [Now](https://zeit.co/now) support
```

Comment:
Could we remove the `Maintained`-part? Happy to merge this but I don't want us to make any claims if a fork is maintained or not 😄.

Reply:
Yes you're right :)